stripe/functions/index.js

```diff
@@ -63,6 +63,22 @@ exports.createStripeCustomer = functions.auth.user().onCreate(event => {
   });
 });
 
+// Add a payment source (card) for a user by writing a stripe payment source token to Realtime database
+exports.addPaymentSource = functions.database.ref('/stripe_customers/{userId}/cards/{token}').onWrite(event => {
```

Comment:
Your original suggestion of listening at `/stripe_customers/{userId}/cards/{pushid}/token` is better. Then writing back makes more sense.